Sources/Models/DetectorType.swift

```diff
@@ -45,6 +44,30 @@ public enum DetectorType {
         case .phoneNumber: return .phoneNumber
         case .url: return .link
         case .transitInformation: return .transitInformation
+        case .custom: return .regularExpression
+        }
+    }
+
+    ///The hashValue of the `DetectorType` so we can conform to `Hashable` and be sorted.
+    public var hashValue: Int {
+        return self.toInt()
+    }
+
+    /// Return an 'Int' value for each `DetectorType` type so `DetectorType` can conform to `Hashable`
+    private func toInt() -> Int {
+        switch self {
+        case .address:
+            return 0
+        case .date:
+            return 1
+        case .phoneNumber:
+            return 2
+        case .url:
+            return 3
+        case .transitInformation:
+            return 4
+        case .custom(let regex):
+            return regex.hashValue
```

Comment:
Nit: Can you put the `return` on the same line as the `case` statement?

Reply:
For sure :)